src/test/java/com/tg5/integration/controller/AccountsControllerTest.java

```diff
@@ -2,22 +2,71 @@
 
 import com.tg5.integration.BaseTest;
 import io.restassured.http.ContentType;
-import org.junit.jupiter.api.Test;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.*;
 
 import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class AccountsControllerTest extends BaseTest {
+    private static Integer createdAccountId;
 
     @Test
-    public void test() {
+    @Order(1)
```

Comment:
Can you tell me about the order? Why is this required? I remember prof saying tests are supposed to be independent. And having an instance variable createdAccountId means you are creating a test sequence. You should set up your data before test and then test it without relying on previous tests.

Reply:
Create test -- Create one account 
Delete/ update test - work on that account 
Otherwise how can they know which account to work on!

Reply:
After the test database should not have any effect. That's why order is defined in a way Create, read, update, delete.